addons/pos_sale/views/product_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">product.template.form.inherit</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
```

Comment:
```suggestion
        <field name="inherit_id" ref="sale.product_template_form_view"/>
```

When possible, try to target the view adding the field you're modifying, to have a cleaner inheritance tree.